tests/test_scripts.py

```diff
@@ -13,4 +13,4 @@
 @pytest.mark.parametrize("test_name", TEST_NAMES)
 def test_script(test_name):
     run = subprocess.run([os.path.join("tests", "scripts", "test_" + test_name + ".sh")])
-    assert run.returncode == 0
+    assert run.returncode in [0, 3]
```

Comment:
```suggestion
    if run.returncode == 3:
        pytest.skip("Skipped as requested by the script.")
    assert run.returncode == 0
```

Reply:
Won't this fail though if the returncode is 3? Maybe we need this:

```python
    if run.returncode == 3:
        pytest.skip("Skipped as requested by the script.")
    else:
        assert run.returncode == 0
```

Reply:
Oh nevermind, I forgot `pytest.skip` halts execution.